default/stringtables/en.txt

```diff
@@ -10901,13 +10901,17 @@ SH_XENTRONIUM
 Xentronium Hull
 
 SH_XENTRONIUM_DESC
-A hull constructed primarily with xentronium. Though small, it has extremely high max [[encyclopedia STRUCTURE_TITLE]]. It has four external slots and one core slot.
+'''A hull constructed primarily with xentronium. Though small, it has extremely high max [[encyclopedia STRUCTURE_TITLE]]. It has four external slots and one core slot.
```

Comment:
Should probably remove all the listings of how many slots are present... This should be shown by the GUI, and is labour-intensive to keep updated.